servicemodels/e2sm_kpm_v2_go/v2/e2sm-kpm-v2-go/builder.go

```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package e2smkpmv2go
+package e2sm_kpm_v2_go
```

Comment:
Package names should not be in form of under_score or micCap. The previous one is good https://go.dev/blog/package-names

Reply:
However, you can remove go at the end.